src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -14,7 +14,7 @@ public function assertResponseOk()
 	{
 		$actual = $this->response->getStatusCode();
 
-		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' .$actual);
+		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' . $actual);
```

Comment:
That document is incomplete though. I work on php-cs-fixer, and changes are made to symfony's official coding standards by us merging cs changes into symfony/symfony. Our code gets used in fabbot.io which checks all symfony pull requests. That's why I said symfony's code is selfdocumenting - because it actually is.